File: 09-13/ColorPicker/ColorPicker/ViewController.m
//
//  ViewController.m
//  ColorPicker
//
//  Created by Artur Lan on 9/13/15.
//  Copyright (c) 2015 Artur Lan. All rights reserved.
//

#import "ViewController.h"
#import <iOS-Color-Picker/FCColorPickerViewController.h>

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIView *color;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)showColorPicker:(UIButton *)sender {
    
    FCColorPickerViewController *vc = [[FCColorPickerViewController alloc] init];
    
    [self.navigationController pushViewController:vc animated:YES];
    
    
}

@end

File: Storm/Storm/ViewController.swift
//
//  ViewController.swift
//  Storm
//
//  Created by Artur Lan on 2/14/16.
//  Copyright © 2016 Artur Lan. All rights reserved.
//

import UIKit
import Foundation

class ViewController: UIViewController {
    
    @IBOutlet weak var cityNameTextField: UITextView!
    @IBOutlet weak var cityNameLabel: UILabel!
    @IBOutlet weak var cityTempLabel: UILabel!
    
    @IBAction func getDataBittonClicked(sender: AnyObject) {
    
    
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        getWeatherData("")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func getWeatherData(urlString : String) {
        
    }
    
    func setLabels(weatherData: NSData) {
        
    }


}

